.github/workflows/ci.yml

```diff
@@ -0,0 +1,150 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# GitHub Actions CI build configuration
+
+name: Ibex CI
+
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+# Note: All tests run as part of one job to avoid copying intermediate build
+# artifacts around (e.g. Verilator and toolchain builds). Once more builds/tests
+# are added, we need to re-evaluate this decision to parallelize jobs and
+# improve end-to-end CI times.
+
+jobs:
+  lint_dv:
+    name: Run quality checks (Lint and DV)
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Fetch all history so that we can run git diff on the base branch
+          fetch-depth: 0
+
+      - name: Setup environment variables
+        run: |
+          # Filter out empty lines or comments
+          grep -v '^\(#\|$\)' ci/vars.env >> $GITHUB_ENV
+
+      - name: Install build dependencies
+        run: |
+          ci/install-build-deps.sh
+
+      - name: Display environment
+        run: |
+          echo $PATH
+          python3 --version
+          echo -n "fusesoc "
+          fusesoc --version
+          verilator --version
+          riscv32-unknown-elf-gcc --version
+          verible-verilog-lint --version
+
+      # Verible format is experimental so only run on default config for now,
+      # will eventually become part of the per-config CI
+      - name: Format all source code with Verible format (experimental)
+        run: |
+          set +e
```

Comment:
Does GitHub's runner set `set -e` where Azure's doesn't?

Reply:
Yes, https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell